nservicebus/messaging/messages-events-commands.md

```diff
@@ -78,3 +79,13 @@ public class MyEvent : IEvent { }
 
 public interface MyEvent : IEvent { }
 ```
+
+
+ ### Conventions
+
+ Conventions are described in the dedicated [Conventions](/nservicebus/messaging/conventions.md) article.
+
+
+ ## Designing Messages
+
+ Messages represent data contract between the endpoints. When creating messages, one should follow the general guidelines for designing contracts, such as the messages should be as small as possible, they should be simple POCO objects, they should follow a Single Responsibility Principle, etc. Classes used for other purposes (e.g. domain objects, data access objects) should not be used as messages.
```

Comment:
"general guidelines for designing contracts"  this is stated like it is a well known approach. when it isnt really. can we link somewhere?